### DISCUSSION TITLE
Hide window on wayland

### DISCUSSION THREAD

**Discussion by CrimsonAge** [emilk/egui#5334] [Q&A]
Is a similar mechanism not possible?I couldn't figure it out(eframe) on egui-sfml hiding viewport works alright
![image](https://github.com/user-attachments/assets/4657035a-74f9-4a18-b13b-c826ade8a24f)

**Comment by mcthesw** [comment]
look at this: https://github.com/emilk/egui/discussions/5127#discussioncomment-12521129
